docs/InterfaceKit.md

```diff
@@ -1,3 +1,135 @@
+|observeOutputs|outputs|Used for asynchronously observing the digital output changes to the Phidget InterfaceKit board. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+|observeInputs|inputs|Used for asynchronously observing the digital input changes to the Phidget InterfaceKit board. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+|observeSensors|sensors|Used for asynchronously observing the analog input changes with an A to D digitalization of 2<sup>10</sup> to the Phidget InterfaceKit board. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+|observeRawSensors|rawSensors|Used for asynchronously observing the analog input changes with an A to D digitalization of 2<sup>12</sup> to the Phidget InterfaceKit board. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+
+##Board
+
+Many different kinds of [Interface Boards](http://www.phidgets.com/products.php?category=0) are available. __board__ will give you info about the features of your board.
+
+|Array Name| Description|
+|---|---|
+|Output|Outputs available|
+|Input|Digital inputs available|
+|Sensor|Analog inputs available, 2<sup>10</sup> digitalization|
+|rawSensor|Analog inputs available, 2<sup>12</sup>digitalization|
+|Trigger|Input triggers available|
+|DataRate|Data rates in milliseconds|
+
+
+
+##Data
+
+|Key|Data Type|Writable|Description|
+|---|---|---|---|
+|type|string|no|'PhidgetInterfaceKit'|
+|dataRateMax|number|no|Maximum data rate in samples per second|
+|dataRateMin|number|no|Minimum data rate in samples per second|
+|dataRate|number|yes|The number of times data is acquired per second|
+|ratiometric|bool|yes| __True__ = configures the Analog Inputs to measure relative to VCC (nominal 5V), __False__ = configures the Analog Inputs to measure relative to an internal precision 5V reference|
+|outputs|array|yes|Outputs of the InterfaceKit|
+|inputs|array|no|Digital inputs of the InterfaceKit|
+|sensors|array|no|Analog inputs of the InterfaceKit with 2<sup>10</sup> digitalization|
+|rawSensors|array|no|Analog inputs of the InterfaceKit with 2<sup>12</sup> digitalization|
```

Comment:
the 2^10 or 2^12 digitalization will not mean anything to the user. see the above comments. this is not in lay terms.